codec/src/main/java/io/netty/handler/codec/ByteToMessageDecoder.java

```diff
@@ -206,7 +206,7 @@ protected ByteBuf internalBuffer() {
     }
 
     @Override
-    public final void handlerRemoved(ChannelHandlerContext ctx) throws Exception {
+    public void handlerRemoved(ChannelHandlerContext ctx) throws Exception {
```

Comment:
@Scottmitch I would love to not do this as this will basically make the `handlerRemoved0` redundant.